### ISSUE TITLE
Bikeshed generation

### ISSUE THREAD

**Issue by lambdapioneer** [AOMediaCodec/av1-avif#7]
I just worked on my first PR and both my local `bikeshed` copy as well as the Curl API https://tabatkins.github.io/bikeshed/#curl generate an `docs/index.html` with a lot of changes vs the status-quo.

If would be great to have either (or both) of the following
 - A reproducible command for local generation (e.g. fixed version)
 - An automated generation (e.g. using CI)

**Comment by cconcolato** [comment]
The command lines I use for local generation is simply
```
cd av1-avif
path_to_bikeshed\bikeshed.py
mv index.html docs\index.html
```

**Comment by lambdapioneer** [comment]
Thanks for the reply @cconcolato – was I was trying to say is that different bikeshed versions will generate different output even when executed with the same command.

That's of course nothing important or urgent, but something that adds friction to contributions :)

**Comment by cconcolato** [comment]
I try to stay up-to-date with the latest version of bikeshed, but you shouldn't worry about that. Worse case I could regenerate the HTML. The source is index.bs.

I thought about using TravisCI to build the spec but it makes it hard to provide a viewable HTML version in pull requests. I could have TravisCI publish the built spec on a separate branch for viewing (I know some groups do that) but that's a bit of an overkill.

BTW, I'm about to change the default branch to use gh-pages and not master, to serve the index.html out of the root folder, not the docs/ folder, like for https://github.com/AOMediaCodec/av1-isobmff.

**Comment by cconcolato** [comment]
I have move the index.html file to the main folder, so that the final `mv` step is not needed. I have also updated the project settings to serve `master` root directory instead of `master/docs`.
I've also updated the readme to indicate that contributions/pull requests should provide the generated HTML as it is easier to view the result. 
Closing this issue as no further action is needed. Reopen if needed.
